<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemView</class>
 <widget class="QDialog" name="MemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory sprite scanner</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="view">
       <property name="minimumSize">
        <size>
         <width>512</width>
         <height>512</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="scrollbar_h">
       <property name="pageStep">
        <number>32</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollBar" name="scrollbar">
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Addr</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="sbWidth">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
         <property name="value">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="sbHeight">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32</number>
         </property>
         <property name="value">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Page</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbPage">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="xHexSpin" name="adrHex">
         <property name="text">
          <string>0000</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cbInvert">
       <property name="text">
        <string>Inversion</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbColumns">
       <property name="text">
        <string>Columns</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbGrid">
       <property name="text">
        <string>Grid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbScreen">
       <property name="text">
        <string>ZX screen</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tbSave">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../xpeccy.qrc">
           <normaloff>:/images/floppy.png</normaloff>:/images/floppy.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xHexSpin</class>
   <extends>QLineEdit</extends>
   <header location="global">stdlib.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sbPage</tabstop>
  <tabstop>adrHex</tabstop>
  <tabstop>sbWidth</tabstop>
  <tabstop>sbHeight</tabstop>
  <tabstop>tbSave</tabstop>
 </tabstops>
 <resources>
  <include location="../xpeccy.qrc"/>
 </resources>
 <connections/>
</ui>
